docs/_getting-started-macos-android.md

```diff
@@ -65,30 +65,30 @@ Next, select the "SDK Tools" tab and check the box next to "Show Package Details
 
 Finally, click "Apply" to download and install the Android SDK and related build tools.
 
-You can also run the following command after setting ANDROID_HOME.
+You can also run the following command after setting ANDROID_SDK_ROOT.
 
 ```shell
 sdkmanager "platforms;android-30" "system-images;android-30;default;x86_64" "system-images;android-30;google_apis;x86"
 sdkmanager "cmdline-tools;latest" "build-tools;30.0.2"
 ```
 
-<h4>3. Configure the ANDROID_HOME environment variable</h4>
+<h4>3. Configure the ANDROID_SDK_ROOT environment variable</h4>
 
 The React Native tools require some environment variables to be set up in order to build apps with native code.
 
 Add the following lines to your `$HOME/.bash_profile` or `$HOME/.bashrc` (if you are using `zsh` then `~/.zprofile` or `~/.zshrc`) config file:
 
 ```shell
-export ANDROID_HOME=$HOME/Library/Android/sdk
-export PATH=$PATH:$ANDROID_HOME/emulator
-export PATH=$PATH:$ANDROID_HOME/tools
-export PATH=$PATH:$ANDROID_HOME/tools/bin
-export PATH=$PATH:$ANDROID_HOME/platform-tools
+export ANDROID_SDK_ROOT=$HOME/Library/Android/sdk
+export PATH=$PATH:$ANDROID_SDK_ROOT/emulator
+# Commandline Path is determined by version "/cmdline-tools/[VERSION]/bin" latest is the default from Android studio but please check you setup within the SDK Manager
+export PATH=$PATH:$ANDROID_SDK_ROOT/cmdline-tools/latest/bin
```

Comment:
Could you revert this as well?

Reply:
I can do however the doc directly references the sdkmanager CLI higher in this file (in lines 71-72). If we remove this we should remove reference to using the SDkManager CLI as well. As above let us know how you want to play it.

Reply:
My bad sorry. I just realized what is the full scope of this change.

I would anyway:
1. Revert this change to align it to the linux one
2. Remove lines 68-73 OR update them to use `$ANDROID_SDK_ROOT/cmdline-tools/latest/bin/sdkmanager` instead of just `sdkmanager`. You would have to add a note **there** that this works only for users that have the `cmdline-tools` package installed.

Reply:
I've removed it. As you say it would be nice to align with the linux docs. 

If anyone wishes to use the CLI they can go to the Android docs which should remain up to date and reduce maintenance on this codebase.